frontend/src/pages/[user]/[project]/versions/new.vue

```diff
@@ -47,7 +47,16 @@ const steps: Step[] = [
     beforeNext: async () => {
       return createPendingVersion();
     },
-    disableNext: computed(() => file.value == null && url.value == null),
+    disableNext: computed(() => {
+      const currentNonNullURLValue = url?.value ?? "";
+      return (
+        file.value == null &&
+        (currentNonNullURLValue === "" ||
+          artifactURLRules.some((v) => {
+            return !v.$validator(currentNonNullURLValue, undefined, undefined);
+          }))
+      );
+    }),
```

Comment:
This needs to accept a file

Reply:
Am I missing functionality here ?
[func.webm](https://user-images.githubusercontent.com/32834385/180431224-4ed09654-2546-420d-8f61-0863af9478d2.webm)

disableNext becomes false if `file.value == null && [...]`, that should cover an uploaded file no?

Reply:
Oh nevermind I was reading it wrong